File: template/src/main/java/ru/surfstudio/standard/interactor/common/error/handler/BaseErrorHandler.java
package ru.surfstudio.standard.interactor.common.error.handler;

import ru.surfstudio.standard.interactor.common.error.HttpProtocolException;

/**
 * Базовый класс обработки ошибок сервера
 */

public interface BaseErrorHandler {

    void handle(HttpProtocolException e);
}

File: template/src/main/java/ru/surfstudio/standard/interactor/common/error/handler/BaseErrorHandler.kt
package ru.surfstudio.standard.interactor.common.error.handler

import ru.surfstudio.standard.interactor.common.error.HttpProtocolException

/**
 * Базовый класс обработки ошибок сервера
 */

interface BaseErrorHandler {

    fun handle(e: HttpProtocolException)
}
